### DISCUSSION TITLE
How to add YouTube channel to Social Integration

### DISCUSSION THREAD

**Discussion by efrenlopezm** [alshedivat/al-folio#637] [Q&A] [answered]
Hello! 
How can I add my YouTube channel to the social integration bar in the same way we can add Google Scholar?
I modified social.html and added:
{% if site.youtube_channel %} ...
But it didn't work.

Thanks!

**Comment by abhinavbharadwajr** [comment]
add the following line in `_config.yml` file under the 'Social Integration' part :

            youtube_channel: <link-to-your-youtube-channel>

and add the below code to `_includes/social.html`

            {%- if site.youtube_channel -%}
            <a href="{{ site.youtube_channel }}" title="YouTube"><i class="fab fa-youtube"></i></a>
            {% endif %}

**Comment by george-gca** [comment]
Currently supported in the repo.

**Reply by richik-p** [reply]
Hi I can't find the social integration in the _config.yml file anymore. @george-gca has this changed?

**Reply by george-gca** [reply, marked as answer]
Yes, we moved all socials to [_data/socials.yml](https://github.com/alshedivat/al-folio/blob/main/_data/socials.yml) to declutter `_config.yml`.
